### ISSUE TITLE
Add Device Shelly UNI

### ISSUE THREAD

**Issue by mschuricht** [windkh/node-red-contrib-shelly#11]
Please include the new Shelly UNI devices!
It Works with Shelly switch Node, but it comes a error message "Shelly type SHUNI-1 is not known"

**Comment by WillyWurst** [comment]
Would be great to have the possibility to interact with new shelly Uni in Node-red with help of this plugin.

**Comment by WillyWurst** [comment]
Just as information: I got the Shelly UNI  running via changing line 86 in 99-shelly.js (https://github.com/windkh/node-red-contrib-shelly/blob/fe598f251686f8ea22ff8b2eeac45654c1d62b2e/shelly/99-shelly.js#L86).I added an or-statement to the if-clause so that the Shelly UNI is identified. The line now looks as follows: `if(node.shellyInfo.type.startsWith("SHPLG-") || node.shellyInfo.type.startsWith("SHSW-") || node.shellyInfo.type.startsWith("SHUNI-")){`
In node-red you just can use the two outputs via the normal set of commands (as for the shelly 2.5 in switch mode). 
I am not familiar with pull requests or how to suggest this change into the deployment pipeline. So sorry for that. But maybe it helps getting Shelly UNI running in node-red as long as the owners of this plugin are going to update it.
